dev/io.openliberty.jakarta.concurrency.3.1_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -21,8 +21,8 @@
 
     <properties>
         <jakarta.concurrent.tck.groupid>jakarta.enterprise.concurrent</jakarta.concurrent.tck.groupid>
-        <jakarta.concurrent.tck.version>3.1.0-SNAPSHOT</jakarta.concurrent.tck.version>
-        <jakarta.concurrent.version>3.1.0</jakarta.concurrent.version>
+        <jakarta.concurrent.tck.version>3.1.1-SNAPSHOT</jakarta.concurrent.tck.version>
```

Comment:
why snapshot instead of just `3.1.1`?

Reply:
I wanted to have a way of quickly switching between SNAPSHOT and release versions for local testing. 
This property is overridden by the test, but if I want to locally test the SNAPSHOT version I just comment out that part of the test.
https://github.com/KyleAure/open-liberty/blob/concurrency-3.1.1/dev/io.openliberty.jakarta.concurrency.3.1_fat_tck/fat/src/io/openliberty/jakarta/enterprise/concurrent/tck/ConcurrentTckLauncherWeb.java#L50-L51

The reason I didn't do this in the reverse order is because there was a time when the TCK was in the Jakarta staging repo for a while and so I uploaded the staged TCK to DHE and instead of using the group `jakarta.enterprise.concurrent` I prefixed it `io.openliberty.jakarta.enterprise.concurrent`.
This it made most sense to have the test override the pom. 

Hope that makes sense 🤣